src/meta/src/stream/source_manager.rs

```diff
@@ -716,13 +722,12 @@ where
             })));
             tracing::debug!("pushing down mutation {:#?}", command);
 
-            tokio_retry::Retry::spawn(FixedInterval::new(Self::SOURCE_RETRY_INTERVAL), || async {
-                let command = command.clone();
-                self.barrier_manager.run_command(command).await
-            })
-            .await
-            .expect("source manager barrier push down failed");
-
+            let (run_command_tx, run_command_rx) = oneshot::channel::<MetaResult<()>>();
+            self.split_diff_tx.send((command, run_command_tx)).unwrap();
+            run_command_rx
```

Comment:
We'd better retry it rather than panic here.

Reply:
I think sending to channel is unlikely to fail. we may retry if run command fails.

Reply:
Yes, that's exactly what I meant.